aospy/region.py

```diff
@@ -1,14 +1,145 @@
+    over region-average quantities.
+
+    Region boundaries are specified as either a single "rectangle" in latitude
+    and longitude or the union of multiple such rectangles.  In addition, a
+    land or ocean mask can be applied.
+
+    Attributes
+    ----------
+    name : str
+        The region's name
+    description : str
+        A description of the region
+    mask_bounds : tuple
+        The coordinates definining the lat-lon rectangle(s) that define(s)
+        the region's boundaries
+    do_land_mask
+        Whether values occurring over land, ocean, or neither are excluded from
+        the region, and whether the included points must be strictly land or
+        ocean or if fractional land/ocean values are included.
+
+    See Also
+    --------
+    aospy.Calc.region_calcs
+
+    """
+
     def __init__(self, name='', description='', lon_bounds=[], lat_bounds=[],
                  mask_bounds=[], do_land_mask=False):
-        """Instantiate a Region object."""
+        """Instantiate a Region object.
+
+        If a region spans across the endpoint of the data's longitude array
+        (i.e.  it crosses the Prime Meridian for data with longitudes spanning
+        0 to 360), it must be defined as the union of two sections extending to
+        the east and to the west of the Prime Meridian.
+
+        Parameters
+        ----------
+        name : str
+             The region's name.  This must be unique from that of any other
+             `Region` objects being used by the overlying `Proj`.
+        description : str, optional
+             A description of the region.  This is not used internally by
+             aospy; it is solely for the user's information.
+        lon_bounds, lat_bounds : length-2 sequence, optional
+             The longitude and latitude bounds of the region.  If the region
+             boundaries are more complicated than a single lat-lon rectangle,
+             use `mask_bounds` instead.
+        mask_bounds : sequence, optional
+             Each element is a length-2 tuple of the format `(lat_bounds,
+             lon_bounds)`, where each of `lat_bounds` and `lon_bounds` are
+             of the form described above.
+        do_land_mask : { False, True, 'ocean', 'strict_land', 'strict_ocean'},
+                       optional
+             Determines what, if any, land mask is applied in addition to the
+             mask defining the region's boundaries.  Default `False`.
+
+             - True: apply the data's full land mask
+             - False: apply no mask
+             - 'ocean': mask out land rather than ocean
+             - 'strict_land': mask out all points that are not 100% land
+             - 'strict_ocean': mask out all points that are not 100% ocean
+
+        Examples
+        --------
+        Define a region spanning the entire globe
+        >>> globe = Region(name='globe', lat_bounds=(-90, 90),
```

Comment:
These examples aren't rendering properly in built docs; I think you might just need a blank line separating the >>> block from the text.